module/gurps.js

```diff
@@ -2514,15 +2518,15 @@ if (!globalThis.GURPS) {
       await handleCombatTurn(combat, turn)
     })
 
-    Hooks.on('deleteCombat', async combat => {
-      console.log(`Combat ended: ${combat.id} - restarting token actions`)
-      await resetTokenActions(combat)
-    })
+    // Hooks.on('deleteCombat', async combat => {
+    //   console.log(`Combat ended: ${combat.id} - restarting token actions`)
+    //   await resetTokenActions(combat)
+    // })
 
-    Hooks.on('deleteCombatant', async (combatant, combat) => {
-      console.log(`Combatant removed: ${combatant.token.name} - resetting token actions`)
-      await resetTokenActionsForCombatant(combatant)
-    })
+    // Hooks.on('deleteCombatant', async (combatant, combat) => {
+    //   console.log(`Combatant removed: ${combatant.token.name} - resetting token actions`)
+    //   await resetTokenActionsForCombatant(combatant)
+    // })
```

Comment:
Make sure we go back and delete this code after we feel confident the new changes worked.